src/common/types/store-data/user-configuration-store.ts

```diff
@@ -9,11 +9,8 @@ export interface UserConfigurationStoreData {
     bugServicePropertiesMap: BugServicePropertiesMap;
 }
 
-interface BugServicePropertiesMap {
+export interface BugServicePropertiesMap {
     [service: string]: BugServiceProperties;
 }
 
-interface BugServiceProperties {
-    [name: string]: string;
-    issueTrackerPath?: string;
-}
+export type BugServiceProperties = Object;
```

Comment:
Is it even worth it anymore to have this type? I guess we could remove it at this point.
I'm curious about the `issueTrackerPath` usage though... I guess we may be using it on our current github implementation.

Reply:
Actually, the type I removed was being used. I changed those instances since they were just assuming those properties would exist. I'm okay with getting rid of this type though. 😄